### PR TITLE
#732 Re-estimate project size

### PR DESCRIPTION
**Pull request by mkordas**
For #732:
* estimation based on guidelines from https://github.com/teamed/est
* estimation has name `002.est` as requested in https://github.com/yegor256/netbout/issues/732#issuecomment-154722733
* `mv first.est 001.est` for consistency
* results: 
    
    ```
Total: 130
2015-05-29: 97 hours by Ivan Inozemtsev
2015-12-18: 164 hours by Michal Kordas
```

### CODE REVIEW

**Comment by karato** [comment]
@mkordas I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by karato** [comment]
@longtimeago review it

File: est/002.est [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  9:
+    worst-case: 30
+    best-case: 10
+    most-likely: 20 
+  8:
+    worst-case: 40
+    best-case: 20
+    most-likely: 30
+  11:
+    worst-case: 50
```
Comment (longtimeago) [inline est/002.est]: @mkordas estimation champions should go thirst

**Comment by longtimeago** [comment]
@mkordas one small remark above

File: est/002.est (hunk 1 above)
Comment (mkordas) [inline est/002.est]: @longtimeago you are right, fixed

**Comment by longtimeago** [comment]
@karato merge

**Comment by longtimeago** [comment]
@mkordas thanks!

**Comment by mkordas** [comment]
@longtimeago 
> @karato merge

Is karato responsible for merging? Did you mean rultor?

**Comment by longtimeago** [comment]
@mkordas oops ... ))

**Comment by longtimeago** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@ypshenychka please, check this issue for QA compliance, as per [par.24](http://at.teamed.io/policy.html#24)

**Comment by karato** [comment]
@rultor deploy pls

**Comment by ypshenychka** [comment]
@longtimeago 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

Only one issue was found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by longtimeago** [comment]
@ypshenychka I confirm

**Comment by ypshenychka** [comment]
@longtimeago thank you

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thanks for the review, we'll try better next time

**Comment by karato** [comment]
@longtimeago added 10 mins to @ypshenychka, for QA review, payment code is `72778584`... Thank you, I have added **22 mins** to your account in payment/transaction "`72778606`", time consumed: 23 hours and 58 mins... extra minutes for review comments (c=7)... added +22 to your rating, now it is equal to [+1315](http://www.netbout.com/b/35024?open=rating)

**Pull request merged**